monitoring/check_pis.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+pswd="piman"
+hosts="../hosts.csv"
+
+
+while read -r line # Read hosts file one line at a time to get ip addresses of pis
+do
+    # Parse the line by replacing ; with spaces, then grabbing the ip address from the resulting array
+    base=${line##*/}
+    parsed_line="${line%/*}/${base//;/ }"
+    array=($parsed_line)
+    ip="${array[1]}"
+
+    # Now check the status of each pi
+    printf "Checking pi@$ip\n"
```

Comment:
it would also be good to take the userid from the command line. perhaps $2 (make sure to add USAGE if $1 or $2 are missing